akka-docs-dev/rst/scala/code/docs/stream/StreamTcpDocSpec.scala

```diff
@@ -25,33 +26,41 @@ class StreamTcpDocSpec extends AkkaSpec {
   // silence sysout
   def println(s: String) = ()
 
-  val localhost = new InetSocketAddress("127.0.0.1", 8888)
-
   "simple server connection" ignore {
```

Comment:
Am I misremembering things or weren't those enabled a while ago?
Didn't track all PRs in last weeks.